### DISCUSSION TITLE
Version 5.1.0 - Drawer is broken

### DISCUSSION THREAD

**Discussion by shanthuk** [ant-design/ant-design#39776] [General]
We are using drawer in our application and we cannot upgrade from 5.0.4 to 5.1.0

https://codesandbox.io/s/basic-antd-5-1-0-forked-9t2vtu?file=/demo.tsx

**Comment by shanthuk** [comment]
https://codesandbox.io/s/basic-antd-5-1-0-forked-9t2vtu?file=/demo.tsx

**Reply by fnoopv** [reply]
![image](https://user-images.githubusercontent.com/24599198/209439832-f7b04b5d-7446-4c55-9830-e7af6c74cef3.png)
placement为什么设置为undefined？只支持上下左右四个方向

**Comment by shanthuk** [comment]
Tell them you are using a wrapper for drawer, sometimes, placement can have an undefined value. We are using Ant Design 5.0.4 and it works correctly. When we tried to upgrade 5.1.0, we are facing this issue. My developer mentioned that a change in CSS (Ant Design) is causing this.
Let me know if you need any more information to resolve this issue. We really want to upgrade to latest Ant Design, but this issue is stopping us.

**Comment by afc163** [comment]
Fixed in https://github.com/react-component/drawer/pull/370

**Comment by shanthuk** [comment]
Thank you. When is the next official release happening?

**Reply by yoyo837** [reply]
Remove `node_modules` and lock file then reinstall it
